File: output/Llama-2-100/avatar/Python/Java/codeforces_459_B.java
import java.util.Scanner;

public class codeforces_459_B {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        String[] l1 = scanner.nextLine().split(" ");
        Arrays.sort(l1);
        int a = l1[-1];
        int b = l1[0];
        if (a == b) {
            System.out.println(a - b + (n * (n - 1)) / 2);
        } else {
            int ac = Arrays.asList(l1).indexOf(a);
            int bc = Arrays.asList(l1).indexOf(b);
            System.out.println(a - b + ac * bc);
        }
    }
}

File: output/Llama-2-100/avatar/Java/Python/codeforces_459_B.py
def codeforces_459_B(N): Scanner = input Scanner = input(N) int A = [int(x) for x in Scanner] Array.sort(A) min = A[0] max = A[-1] mins = 0 maxs = 0 for x in A: if x == min: mins += 1 if x == max: maxs += 1 if min == max: print((max - min) + " " + mins * (mins - 1) / 2) else: print((max - min) + " " + mins * maxs) return

